pepper-apis/studybuilder-cli/studies/singular/activities/consent-self.conf

```diff
@@ -2289,7 +2289,7 @@
           "bodyTemplate": {
             "templateType": "HTML",
             "templateText": """
-              <p class="paragraph">$ddp.date("MM / dd / yyyy")</p>
+              <p class="paragraph">$ddp.submissionDate("MM / dd / yyyy", $ddp.date("MM / dd / yyyy"))</p>
```

Comment:
Let's not change this file. We will need to make the change to consent form via a patch.

Reply:
Fixed in 61a2b9c100a9c06ee6f8727d30d1e1c5b26ca9ac

Reply:
Code looks reasonable and appears to be working. Could we scrutinize further regarding how these changes would affect studies that are using the snapshot feature , i.e.,. that have activities with the snapshotSubstitutionsOnSubmit=true flag .
I BELIEVE it is OK, but would welcome a closer look with the users of the snapshot functionality in mind.